app/models/user.rb

```diff
@@ -1,9 +1,41 @@
 # frozen_string_literal: true
 
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :bigint           not null, primary key
+#  allow_password_change  :boolean          default(FALSE)
+#  email                  :string           not null
+#  encrypted_password     :string           default(""), not null
+#  gender                 :integer
+#  image                  :string
+#  lastname               :string
+#  name                   :string
+#  nickname               :string
+#  provider               :string           default("email"), not null
+#  reset_password_sent_at :datetime
+#  reset_password_token   :string
+#  tokens                 :json
+#  uid                    :string           default(""), not null
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+# Indexes
+#
+#  index_users_on_email                 (email) UNIQUE
+#  index_users_on_reset_password_token  (reset_password_token) UNIQUE
+#  index_users_on_uid_and_provider      (uid,provider) UNIQUE
+#
 class User < ActiveRecord::Base
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   include DeviseTokenAuth::Concerns::User
+
+  enum gender: { female: 0, male: 1, fluid: 2 }
```

Comment:
it's also possible to use a string column to persist the enumerated value
```ruby
enum gender: { female: 'female', male: 'male', fluid: 'fluid' }
```

Reply:
I prefer to use with `int`, but this way has some advantages?

Reply:
Not really. Using strings has the benefit that the column has readable information